src/components/ContractForm.js

```diff
@@ -99,12 +102,11 @@ class ContractForm extends Component {
     }
 
     fabTooltip = () => {
-        const { rights } = this.props;
-        if (rights.includes(RIGHT_POLICYHOLDERCONTRACT_SUBMIT) && this.isUpdatable()) {
+        if (this.isUpdatable()) {
             return formatMessage(this.props.intl, "contract", "submitButton.tooltip");
-        } else if (rights.includes(RIGHT_POLICYHOLDERCONTRACT_APPROVE) && this.isApprovable()) {
+        } else if (this.isApprovable()) {
             return formatMessage(this.props.intl, "contract", "approveButton.tooltip");
-        } else if (rights.includes(RIGHT_POLICYHOLDERCONTRACT_AMEND) && !this.isUpdatable() && !this.isApprovable() && !this.isTerminated()) {
+        } else if (!this.isUpdatable() && !this.isApprovable() && !this.isTerminated()) {
```

Comment:
'!this.isUpdatable() && !this.isApprovable()' is already true from previous statements.